server/package.json

```diff
@@ -18,7 +19,7 @@
     "@sentry/node": "^6.2.5",
     "@sentry/tracing": "^6.2.5",
     "axios": "^0.21.2",
-    "brave-alert-lib": "https://github.com/bravetechnologycoop/brave-alert-lib#v6.8.0",
+    "brave-alert-lib": "https://github.com/bravetechnologycoop/brave-alert-lib#td-multiple-responders",
```

Comment:
Update after https://github.com/bravetechnologycoop/brave-alert-lib/pull/45 is merged

Reply:
Done.